core/src/cg/group4/game_logic/stroll/Stroll.java

```diff
@@ -113,7 +116,17 @@ protected final void generatePossibleEvent() {
         Random rnd = new Random();
         if (rnd.nextFloat() < cEventThreshold) {
             cEventGoing = true;
-            cEvent = new TestStrollEvent();
+            int chosenEvent = rnd.nextInt(2);
```

Comment:
Couldn't you make this more dynamic by storing it in a list.. Maybe with the help of enums